tests/codeception/acceptance/ComponentsCest.php

```diff
@@ -279,4 +279,15 @@ public function testQuote(AcceptanceTester $I) {
     $I->canSee('alt', '#content');
   }
 
+  /**
+   * Skiplink Component.
+   */
+  public function testSkiplink(AcceptanceTester $I) {
```

Comment:
You probably don't need a new test, just need to alter the existing test. https://github.com/SU-SWS/jumpstart_ui/blob/8.x-1.x/tests/src/Unit/Plugin/Block/SkipNavAnchorBlockTest.php#L81